test/test-debuglet.ts

```diff
@@ -1333,3 +1362,17 @@ describe('Debuglet', () => {
     });
   });
 });
+
+// a counter for unique test urls.
+let apiUrlInc = 0;
+
+/**
+ * returns a new config object to be passed to debuglet. always has apiUrl
+ * @param conf custom config values
+ */
+function debugletConfig(conf?: {}): (ResolvedDebugAgentConfig & {apiUrl:string}) {
+  const apiUrl = API + (++apiUrlInc);
```

Comment:
nit: Move `API` to be a local variable here. Better, it would be nice if we didn't duplicate the constant here and it was exported from `controller.ts`. That doesn't have to be done as part of this change though.

Reply:
i would like to see all api urls/defaults come from config.ts or a consistent place instead of the implementation. 

good call moving it into the function for now it'll make it harder just to use API in a new test instead of the unique name,

Reply:
sgtm